### PR TITLE
Chore: Button ODS APP implementation 

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

Task: [APP-2753](https://aragonassociation.atlassian.net/browse/APP-2753)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2753]: https://aragonassociation.atlassian.net/browse/APP-2753?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/selectChainForm/index.tsx [hunk 1]
```diff
@@ -34,24 +35,22 @@ const SelectChainForm: React.FC = () => {
     <>
       <Header>
         <NetworkTypeSwitcher>
-          <ButtonText
-            mode="ghost"
-            bgWhite
-            size={isMobile ? 'small' : 'medium'}
-            label={t('labels.mainNet')}
-            isActive={networkType === 'main'}
```
Comment (jamesej) [inline src/containers/selectChainForm/index.tsx:42]: Doesn't look like the isActive prop is mapped onto the new ODS Button

File: src/containers/selectChainForm/index.tsx [hunk 2]
```diff
@@ -34,24 +35,22 @@ const SelectChainForm: React.FC = () => {
...
             onClick={() => {
               setNetworkType('main');
             }}
-          />
-          <ButtonText
-            mode="ghost"
-            bgWhite
-            size={isMobile ? 'small' : 'medium'}
-            label={t('labels.testNet')}
-            isActive={networkType === 'test'}
```
Comment (jamesej) [inline src/containers/selectChainForm/index.tsx:52]: Same issue with isActive

File: src/containers/daoFilterModal/index.tsx [hunk 3]
```diff
@@ -78,20 +78,18 @@ const Header: React.FC<HeaderProps> = ({onClose}) => {
...
-        bgWhite
+        variant="secondary"
+        size="sm"
         onClick={onClose}
       />
-      <ButtonIcon
-        icon={<Icon icon={IconType.CLOSE} />}
+      <Button
+        iconLeft={IconType.CLOSE}
         className="hidden lg:block"
```
Comment (jamesej) [inline src/containers/daoFilterModal/index.tsx]: should be lg:flex - lg;block removes flex style and breaks layout of icon in button

**Review by jamesej — CHANGES_REQUESTED** [review]
Hi Rakesh, thanks for grinding through this, pretty much all looks good except for the styling fix I have in a comment. Also it seems that the `bgWhite` and `active` props are just ignored on the new button, can you confirm that's intended?

**Comment by jamesej** [comment]
Another thing, there are buttons in SearchHeader.tsx, stateEmpty/index.tsx and pageEmptyState/index.tsx not yet converted and I think you may have missed converting ButtonWallet.tsx - was that intentional?

**Comment by RakeshUP** [comment]
Hi @jamesej, thanks for the quick review!

- The new ODS button does not have both `isActive` or `bgWhite` prop. Also the `secondary` and `ghost` variant's styles have changed. So the prop changes were intended.
- The new ODS library does not include ButtonGroup and ButtonWallet yet and there are no plans to build these components on the ODS for now. So these changes have to wait.
- Apart from that I've skipped 6 files because they are ODS module components like WYISWYG editor/EmptyState which will anyways be changed soon.

I'll give the whole changes a quick QA and ensure that none of the buttons are breaking the app layout now

**Review by jamesej — APPROVED** [review]
All good, what a giant PR!

**Pull request merged**